docs/usage/self-hosted-configuration.md

```diff
@@ -193,6 +193,13 @@ It will also override any settings in `packageRules`.
 ## forkMode
 
 You probably have no need for this option - it is an experimental setting for the Renovate hosted GitHub App.
+However, if this is set to true then Renovate will clone the repository before working on the clone and creating Pull Requests on the parent (GitHub only).
+
+## forkToken
```

Comment:
Needs to be added to lib/config/definitions.ts too

Reply:
Done